tests/app/v2/notifications/test_post_letter_notifications.py

```diff
@@ -450,15 +485,34 @@ def test_post_letter_notification_is_delivered_and_has_pdf_uploaded_to_test_lett
     s3mock.assert_called_once_with(ANY, b'letter-content', precompiled=True)
 
 
-def test_post_letter_notification_persists_notification_reply_to_text(
+def test_post_letter_notification_ignores_reply_to_text_for_service(
+    client, notify_db_session, mocker
+):
+    mocker.patch('app.celery.letters_pdf_tasks.create_letters_pdf.apply_async')
+
+    service = create_service(service_permissions=[LETTER_TYPE])
+    create_letter_contact(service=service, contact_block='ignored', is_default=True)
+    template = create_template(service=service, template_type='letter')
```

Comment:
To clarify, this test isn't testing when the contact block has no placeholders, it testing that we didn't provide  `reply_to=letter_contact_block.id` in the template?